src/Components/Tailwind/Table.tsx

```diff
@@ -271,19 +273,14 @@ function Row({fullName, checkedIn, state, onClick, isEven}: RowProps) {
     >
       <td className="max-w-0 p-4">
         <div className="flex items-center justify-between">
-          <Typography
-            variant="body1"
-            className={
-              state === 'rejected' || state === 'withdrawn'
-                ? `${fullNameClass} line-through`
-                : fullNameClass
-            }
-          >
+          <Typography variant="body1" className={fullNameClass}>
             {fullName}
           </Typography>
           <div className="flex">
             {state === 'pending' && <ExclamationCircleIcon className="h-6 w-6 text-yellow-500" />}
```

Comment:
This is unrelated to your PR, but I think this makes it a tad more clear for our pending registrants :)

```suggestion
            {state === 'pending' && <ClockIcon className="h-6 w-6 text-yellow-500" />}
```